content/blog/introducing-the-harbor-operator.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Introducing the Harbor Operator"
+author:
+  name: "Harbor Team"
+description: "Simplify the installation and Day 2 operations of Harbor with the Harbor Operator"
+date: 2020-03-23T12:00:00+04:00
+---
+
+## Introducing the Harbor Operator
```

Comment:
This is unnecessary (the title is already displayed at the top)

Reply:
Fixed!